assets/pages/new-contributions-landing/components/ContributionAmount.jsx

```diff
@@ -23,21 +23,23 @@ type PropTypes = {
   currency: IsoCurrency,
   contributionType: Contrib,
   amount: Amount | null,
-  showOther: boolean,
-  selectAmount: Amount => (() => void),
-  selectOtherAmount: () => void,
+  selectedAmounts: { [Contrib]: number },
```

Comment:
I'm finding the name of this confusing. Surely there's only one selected amount at any one time? Or does it serve a different function?

Reply:
For the view, there are three: one-off, monthly and regular and the correct one is chosen depending on the selected contribution type. (this is only useful to restore the previously selected amount when someone changes from one-off to monthly and back for example)

Reply:
aha of course